### DISCUSSION TITLE
параметр hostlist

### DISCUSSION THREAD

**Discussion by muzzy124** [hufrea/byedpi#97] [Q&A] [answered]
список хостов какой формат имеет?
поддерживаются ли шаблоны, * и т.п.?
субдомены автоматически добавляются?

**Comment by hufrea** [comment, marked as answer]
> список хостов какой формат имеет?

текст, содержащий домены разделенные новой строкой или пробелом.

>  поддерживаются ли шаблоны, * и т.п.?

нет.

>  субдомены автоматически добавляются?

Да

**Reply by SReader455** [reply]
А этот файл список где должен лежать?

**Reply by vehametra** [reply]
от текущией папки.
